W-21558980: fixing link (#405)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-config-ldap-pce.adoc

* xref:access-management::managing-users.adoc[Configure SAML]
* xref:access-management::conf-openid-connect-task.adoc[Configure OpenID Connect]
+

> Troubleshooting information can be found at https://help.salesforce.com/s/articleView?id=001114931&type=1[Anypoint Platform Single Sign-On (SSO) using OpenID Connect (OIDC) Troubleshooting Guide].

Author: ```suggestion
For troubleshooting information, see https://help.salesforce.com/s/articleView?id=001114931&type=1[Anypoint Platform Single Sign-On (SSO) using OpenID Connect (OIDC) Troubleshooting Guide].
```
